pyproject.toml

```diff
@@ -12,7 +12,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3"
```

Comment:
What's this about? We don't support python < 3.6.

Reply:
Not related to this PR. I'll revert it.